ui.frontend/src/utils.js

```diff
@@ -231,6 +230,25 @@ export default class Utils {
             (button) => button.addEventListener('click', handler));
     }
 
+    /**
+     * Registers custom functions from clientlibs
+     * @param formId
+     */
+    static async registerCustomFunctions(formId) {
+        const funcConfig = await HTTPAPILayer.getCustomFunctionConfig(formId);
+        if (funcConfig && funcConfig.customFunction){
+            const funcObj = funcConfig.customFunction.reduce((accumulator, func) => {
+                if (window[func.id]) {
+                    accumulator[func.id] = window[func.id];
+                }
+                return accumulator;
+            }, {});
+            FunctionRuntime.registerFunctions(funcObj);
+        } else {
+            console.debug("Fetched custom functions: " + JSON.stringify(funcConfig));
```

Comment:
In case of error (ie old far release), funcConfig would be null. This might break